modules/oe_theme_helper/src/Plugin/Block/PageHeaderBlock.php

```diff
@@ -77,14 +101,15 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function build(): array {
     $metadata = $this->getContext('page_header')->getContextData()->getValue();
-
+    $title = $metadata['title'] ?? $this->title;
```

Comment:
Why did you move this to a new line?

Reply:
@imanoleguskiza ?

Reply:
I need the $title in two places: the build array itself and to generate the breadcrumb array

Reply:
OK!